src/fetch-event-source/fetch.ts

```diff
@@ -128,15 +128,17 @@ export function fetchEventSource(
         ])) as Response;
 
         if (response.status === 404) {
-          throw new ReapError('Channel reaped');
+          onerror?.(new ReapError('Channel reaped'));
```

Comment:
Wondering if we even should be calling the developer-provided `onerror` here. If it got reaped, we will try and seamlessly reestablish a new channel. What is there for the developer to do?

Reply:
this is a little weird because the airlock is the "developer" here. this is not the `onError` passed in to the airlock, but something the airlock itself is passing to handle errors itself. The consumer's `onError` is only called here: https://github.com/urbit/js-http-api/blob/3e17850bbed0f9a4718a5058d911acbb892da489/src/Urbit.ts#L417